ee/maintained-apps/README.md

```diff
@@ -66,3 +66,23 @@ These are command lines that will be run _before_ the generated uninstall script
 
 #### `post_uninstall_scripts`
 These are command lines that will be run _after_ the generated uninstall script is executed.
+
+### Testing
+
+Fleet tests every change to supported Fleet-maintained apps (e.g. new version, updated install script, etc.) and all new apps.
+
+For new apps:
+
+1. When a pull request is opened in `inputs/`, the [#g-software Engineering Manager (EM)](https://fleetdm.com/handbook/company/product-groups#software-group) is automatically added as reviewer.
+2. The EM is responsible for making sure that the `name` for the new app matches the name that shows up in Fleet's software inventory. If the name doesn't match or if the name is not user-friendly, the EM will bring it to #g-software design review. This way, when the app is added to Fleet, the app will be matched with the app that comes back in software inventory.
+3. Then, the EM merges the PR. This automatically opens a PR to `outputs/`
+4. Next, follow steps under "For changes to supported apps." 
+
+For changes to supported apps:
+
+1. A pull request (PR) is automatically opened in `outputs/` and a Product Designer is automatically added as reviwer.
```

Comment:
FYI [this thread](https://fleetdm.slack.com/archives/C086V2QK76X/p1745429228859699) suggests a more surgical approach to existing app updates, with different initial DRIs (the current PR assigneeds) to both reduce bus factor and allow for confidently no-op'ing a significant chunk of testing for changes that are low risk.